docs/web-player.md

```diff
@@ -114,15 +114,16 @@ See the general [Usage](#usage) section above.
 
 #### Attributes
 
-| Attribute    | Property     | Type      | Default | Description                                                                                                                                         |
-| ------------ | ------------ | --------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `bubbles`    | `bubbles`    | `boolean` | `false` | If true then events dispatched by this element will bubble.                                                                                         |
-| `configtype` | `configType` | `string`  | `null`  | Type of stream config to use. Defaults to SDK platform (WEB) when value is null (attribute is absent).                                              |
-| `fullscreen` | `fullscreen` | `boolean` | `false` | If true then player is fullscreen.<br />Dispatches: `livery-fullscreen-change`                                                                      |
-| `loglevel`   | `logLevel`   | `string`  | `null`  | Log level (`'quiet'` \| `'error'` \| `'warn'` \| `'info'` \| `'debug'` \| `'spam'`). Defaults to `'info'` when value is null (attribute is absent). |
-| `muted`      | `muted`      | `boolean` | `false` | If `true` then audio is muted.<br />Dispatches: `livery-volume-change`                                                                              |
-| `streamid`   | `streamId`   | `string`  | `null`  | Livery stream ID (required).                                                                                                                        |
-| `vumeter`    | `vuMeter`    | `boolean` | `false` | Enable volume unit meter.                                                                                                                           |
+| Attribute     | Property      | Type      | Default | Description                                                                                                                                         |
+| ------------- | ------------- | --------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `akamaitoken` | `akamaiToken` | `string`  | `null`  | Authentication token to use to be able to see a stream that uses Akamai’s Token Auth feature.                                                       |
```

Comment:
Should this read 'Access token ...' rather than 'Authentication token ...' ?

cf. https://techdocs.akamai.com/adaptive-media-delivery/docs/add-token-auth

Reply:
I discussed this with PP I think.
We were thinking authentication token as the feature is called token authentication.
However, looking at the documentation again it seems they are calling the tokens access tokens indeed.

Reply:
Yes. There are two different tokens involved in token authentication and this refers to only one of them

Reply:
Ah, I didn't really think about that at the time. Looking at this now I see as part of the Token Authentication there are two types of tokens used: an access token and a session token. We could indeed improve this documentation to read: "Access token to use to be able to see a stream that uses Akamai’s Token Auth feature.". Note that in that case we should also correspondingly update the jsdoc for the property in the player-web repo. Shall I do that?

Reply:
Yes. Please do

Reply:
Done, see also: https://github.com/liveryvideo/player-web/commit/215ad1c7509014125ac859092dc8eddc5b3cabb9